.github/workflows/ci.yml

```diff
@@ -2,8 +2,8 @@ name: "sandbox ci"
 
 env:
   GO_VERSION: 1.16
-  MINIKUBE_VERSION: v1.20.0
-  KUBERNETES_VERSION: v1.20.4
+  MINIKUBE_VERSION: v1.24.0
+  KUBERNETES_VERSION: v1.20.7
```

Comment:
what issue(s) the kubernetes version resolved while updating the minikube version?

Reply:
Now it maches exactly to what we have on EKS